api/assets/schemas.json

```diff
@@ -12499,6 +12499,11 @@
                 "maxLength": 100,
                 "type": "string"
             },
+            "discriminator": {
+                "minLength": 4,
+                "maxLength": 4,
```

Comment:
make this observe the existing config item on discriminator length

Reply:
> make this observe the existing config item on discriminator length

Oh, I forgot to re-generate the schema after removing those, they shouldn't be needed as it's already checked in the User entity afaik.